Rewrite scan20 overview (#2717)
pivotal/docs-tap: scst-scan/app-scanning-beta.hbs.md

## <a id="overview"></a>Overview

> SCST - Scan 2.0 is responsible for providing the framework to scan components of workloads to help users increase the security posture of their application.  The initial capability for this component is to provide the ability to scan container images using a users prefered container image scan solution for known Common Vulnerabilities and Exposures (CVEs).  By including this component in a users software supply chain, a user can identify vulnerabilites earlier in the development lifecycle. 

Reviewer: ```suggestion
The initial capability for this component is to provide the ability to scan container images using a users preferred container image scan solution for known Common Vulnerabilities and Exposures (CVEs).  By including this component in a users software supply chain, a user can identify vulnerabilities earlier in the development lifecycle. 
```

> The Scan 2.0 simplifies integrating image scan solutions such as Anchore's Grype, Aqua's Trivy, Palo Alto's Prisma, and VMware's Carbon Black Cloud in to the Tanzu Application Platform.  This is achieved by the following enhancements over Scan 1.0:

Reviewer: ```suggestion
The Scan 2.0 simplifies integrating image scan solutions such as Anchore's Grype, Aqua's Trivy, Palo Alto's Prisma, and VMware's Carbon Black Cloud into the Tanzu Application Platform. This is achieved by the following enhancements over Scan 1.0:
```

> * Reducing the scope of what image scanning is resposible to only be 1) scanning a container image and 2) pushing the scan results in an industry standard format ([CycloneDX](https://cyclonedx.org/) or [SPDX](https://spdx.dev/) as an industry standard OCI artifact to an OCI compliant container registry.  Using industry standards mitigates the need for users wishing to create an integration to be familar with properitary Tanzu Application Platform configurations.

Reviewer: ```suggestion
* Reducing the scope of what image scanning is responsible to only be:
** Scanning a container image.
** Pushing the scan results in an industry standard format ([CycloneDX](https://cyclonedx.org/) or [SPDX](https://spdx.dev/) as an industry standard OCI artifact to an OCI compliant container registry.  Using industry standards mitigates the need for users wishing to create an integration to be familiar with proprietary Tanzu Application Platform configurations.
```
